lerobot/common/datasets/lerobot_dataset.py

```diff
@@ -126,6 +128,9 @@ def tolerance_s(self) -> float:
         are not close enough from the requested frames. It is only used when `delta_timestamps`
         is provided or when loading video frames from mp4 files.
         """
+        if self.tolerance_s is not None:
```

Comment:
I think the "right" way to handle this would be to have a private attribute `self._tolerance` which you set to `1 / fps - 1e-4` in the `__init__` (btw `fps` should also be private as in `self._fps` and then might need its own getter method). Then here you just need to return `self._tolerance`. 

OR, if we don't want to bother with encapsulation, this getter should disappear and we should just have an attribute `self.tolerance_s`. The only danger here is that it's independent of the `fps` from an interface perspective.

@Cadene I think you should weigh in here as I know you're not a fan of using private attributes.

Reply:
@villekuosmanen After discussing internally, we will put your PR on hold for a bit while we think about a design.
cc @alexander-soare

Reply:
@villekuosmanen yeah so since I am encountering this right now with some RL stuff I am doing, we just want to get wait that working then use whatever it was I did to make it happen. It might be this approach, but there may be other ways of handling it.

Thanks a lot mate!

Reply:
yep no problem, happy to wait.